src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="search-form"
+      action="/"
+      method="get"
+    >
+      <label class="search-input">
```

Comment:
Because it's not the input

```suggestion
      <label class="search-input-wrapper">
```